src/components/custom-field-input-multiple-choice/custom-field-input-multiple-choice.jsx

```diff
@@ -178,6 +179,9 @@ function CustomFieldInputMultipleChoice(props) {
                 {choice.label}
               </ListOption>
             ))}
+            {visibleChoices.length === 0 && (
+              <NoChoiceListItem noOptionText={props.noOptionText} />
```

Comment:
If the "empty state" is not considered a list item then we should not render it in a `Listbox`. We will want to come up with a "popup" component that can render arbitrary components within it. 

To be concrete, it will look something like this:

```
{renderPopup && (
  <Popup>
    {visibleChoices.length === 0 ? (
      <EmptyState text="wuuuuut" /> 
    ) : ( 
      <Listbox>
        ...
      </Listbox>
    )}
  </Popup>
)}